querybook/server/env.py

```diff
@@ -83,6 +83,8 @@ class QuerybookSettings(object):
     STORE_MAX_UPLOAD_CHUNK_NUM = int(get_env_config("STORE_MAX_UPLOAD_CHUNK_NUM"))
     STORE_MAX_READ_SIZE = int(get_env_config("STORE_MAX_READ_SIZE"))
     STORE_READ_SIZE = int(get_env_config("STORE_READ_SIZE"))
+    STORE_BUCKET_S3V4_ENABLED = get_env_config("STORE_BUCKET_S3V4_ENABLED")
```

Comment:
since STORE_... are kind of shared between GCP and AWS, can you prefix the setting with something like S3? It can be `S3_BUCKET_V3_ENABLED`.

Also since the value is boolean, lets do
```
    STORE_BUCKET_S3V4_ENABLED = get_env_config("STORE_BUCKET_S3V4_ENABLED") == "true"
```
so in the code we can use it as boolean directly

Reply:
Done.